### PR TITLE
EffectSsStone1 Ok

### PR DESCRIPTION
**Pull request by hensldm**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: assets/xml/objects/gameplay_keep.xml [hunk 1]
```diff
@@ -1050,15 +1050,15 @@
...
-        <DList Name="gameplay_keep_DL_03C190" Offset="0x3C190" />
+        <Texture Name="gEffStone8Tex" OutName="eff_stone_8" Format="ia8" Width="32" Height="32" Offset="0x3A190" />
+        <Texture Name="gEffStone7Tex" OutName="eff_stone_7" Format="ia8" Width="32" Height="32" Offset="0x3A590" />
+        <Texture Name="gEffStone6Tex" OutName="eff_stone_6" Format="ia8" Width="32" Height="32" Offset="0x3A990" />
+        <Texture Name="gEffStone5Tex" OutName="eff_stone_5" Format="ia8" Width="32" Height="32" Offset="0x3AD90" />
+        <Texture Name="gEffStone4Tex" OutName="eff_stone_4" Format="ia8" Width="32" Height="32" Offset="0x3B190" />
+        <Texture Name="gEffStone3Tex" OutName="eff_stone_3" Format="ia8" Width="32" Height="32" Offset="0x3B590" />
+        <Texture Name="gEffStone2Tex" OutName="eff_stone_2" Format="ia8" Width="32" Height="32" Offset="0x3B990" />
+        <Texture Name="gEffStone1Tex" OutName="eff_stone_1" Format="ia8" Width="32" Height="32" Offset="0x3BD90" />
+        <DList Name="gEffStoneDL" Offset="0x3C190" />
```
Comment (AngheloAlf) [inline assets/xml/objects/gameplay_keep.xml:1061]: Why the textures are named in descending order?

Comment (hensldm) [reply assets/xml/objects/gameplay_keep.xml:1061]: For some reason I thought this was the way OOT had them. This is not the case, so will reverse the order.

**Comment by EllipticEllipsis** [comment]
I assumed it was due to how they're laid out in the DrawInfo.

**Review by AngheloAlf — APPROVED** [review]

**Comment by hensldm** [comment]
Just a note, cleaned up `undefined_syms` based on this overlay (just 1 removal).

**Review by AngheloAlf — APPROVED** [review]

File: undefined_syms.txt [hunk 2]
```diff
@@ -430,7 +430,6 @@ D_040367B0 = 0x040367B0;
 D_040377B0 = 0x040377B0;
 D_04037850 = 0x04037850;
 D_0403A0F0 = 0x0403A0F0;
-D_0403C190 = 0x0403C190;
```
Comment (EllipticEllipsis) [inline undefined_syms.txt:433]: I'm sot sure it's worth doing this until we can just remove them all at once.

**Review by EllipticEllipsis — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**